homeassistant/helpers/config_validation.py

```diff
@@ -219,6 +221,17 @@ def validate(obj):
     return validate
 
 
+def validate_values(valid_values):
```

Comment:
If it gets more involved than the vol.all/coerce/in test your new function should definitely be used in other places to avoid mistakes.